tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -0,0 +1,252 @@
+        IF    "${workbench_cr_name}" == "${EMPTY}"
+            IF    "${namespace}" == "${EMPTY}"
+                ${namespace}=        Get Openshift Namespace From Data Science Project    project_title=${project_title}
+            END
+                ${_}    ${workbench_cr_name}=       Get Openshift Notebook CR From Workbench    workbench_title=${workbench_title}    namespace=${namespace}
+        END
+        Click Element       xpath=//tr[td/div/a[text()="${project_title}"]]/td/div//label[@for="${workbench_cr_name}"]/span[@class="pf-c-switch__toggle"]
+        Handle Stop Workbench Confirmation Modal    press_cancel=${press_cancel}
+    ELSE
+        Fail     msg=Cannot stop ${workbench_title} workbench because it is not running.
+    END
+    Wait Until Workbench Is Stopped From Projects Home Page     workbench_title=${workbench_title}
+    ...                                                         project_title=${project_title}
+
+Wait Until Workbench Is Started From Projects Home Page
+    [Documentation]    Waits until the workbench status is "RUNNING", from the DS Project home page
+    [Arguments]     ${workbench_title}   ${project_title}   ${timeout}=30s
+    Wait Until Keyword Succeeds    ${timeout}    5s      Workbench Status Should Be From Projects Home Page
+    ...     workbench_title=${workbench_title}   status=${WORKBENCH_STATUS_RUNNING}   project_title=${project_title}
+
+Wait Until Workbench Is Stopped From Projects Home Page
+    [Documentation]    Waits until the workbench status is "STOPPED", from the DS Project home page
+    [Arguments]     ${workbench_title}   ${project_title}   ${timeout}=30s
+    Wait Until Keyword Succeeds    ${timeout}    5s      Workbench Status Should Be From Projects Home Page
+    ...    workbench_title=${workbench_title}   status=${WORKBENCH_STATUS_STOPPED}   project_title=${project_title}
+
+Launch Workbench From Projects Home Page
+    [Documentation]    Launches a running workbench from DS Projects home page.
+    [Arguments]     ${workbench_title}    ${project_title}
+    ${is_started}=      Run Keyword And Return Status   Workbench Status Should Be From Projects Home Page
+    ...    workbench_title=${workbench_title}   status=${WORKBENCH_STATUS_RUNNING}    project_title=${project_title}
+    IF    ${is_started} == ${TRUE}
+        Click Element       xpath=//tr[td/div/a[text()="${project_title}"]]/td//a[text()="${workbench_title}"]
+        Access To Workbench
+    ELSE
+        Fail   msg=Cannot launch workbench ${workbench_title} because it is not running...
+    END
+
+Workbench Launch Link Should Be Disabled
+    [Documentation]    Checks the workbench launch link is disabled from DS Projects home page.
+    [Arguments]     ${workbench_title}    ${project_title}
+    ${link_status}=    Get Element Attribute    xpath=//tr[td/div/a[text()="${project_title}"]]/td//a[text()="${workbench_title}"]    aria-disabled
+    Should Be Equal As Strings    ${link_status}    true
```

Comment:
If you add `msg=Workbench launch link should be disabled for workbench ${workbench_title} in project ${project_title}` the console log will be easier to understand

Reply:
makes sense, done